src/main/java/com/flop/resttester/components/keyvaluelist/KeyValuePair.java

```diff
@@ -13,27 +13,30 @@ public class KeyValuePair implements Cloneable {
 
     public String key;
     public String value;
+    public Boolean enabled;
 
-    public KeyValuePair(String key, String value) {
+    public KeyValuePair(String key, String value, Boolean enabled) {
         this.key = key;
         this.value = value;
+        this.enabled = enabled;
     }
 
     public JsonObject getAsJson() {
         JsonObject jObj = new JsonObject();
         jObj.addProperty("key", this.key);
         jObj.addProperty("value", this.value);
+        jObj.addProperty("enabled", this.enabled);
         return jObj;
     }
 
     public static KeyValuePair createFromJson(JsonObject jObj) {
         if (!jObj.has("key") && !jObj.has("value")) {
             throw new RuntimeException("Invalid key value object. Key or value property not found.");
         }
-        return new KeyValuePair(jObj.get("key").getAsString(), jObj.get("value").getAsString());
+        return new KeyValuePair(jObj.get("key").getAsString(), jObj.get("value").getAsString(), jObj.get("enabled").getAsBoolean());
```

Comment:
This line is a bit problematic. If a user is currently using the addon and has some key value pairs already stored, the "enabled" property is null. So after he updates to the new version this will throw a null pointer exception.

So we need to add a fallback here. E.g. create a bool value which is true by default and if the jObj.get("enable") is not null update the value with the stored value

Reply:
Good point. I hadn't considered upgrade compatibility issues. 

Updated to set the value to `true` if the json property is null (via short-circuiting `||`), or the boolean value if present.